src/server.rs

```diff
@@ -64,6 +65,17 @@ pub struct InclusionProofRequest {
     pub identity_commitment: Hash,
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[serde(deny_unknown_fields)]
+pub struct VerifyProofRequest {
```

Comment:
It is definitely worth documenting that this endpoint corresponds to the verification of _semaphore_ protocol proofs, and not the merkle tree (insertion/update/removal) proofs. Probably worth renaming that to be clear.